core/storage/story/gae_models_test.py

```diff
@@ -106,6 +151,18 @@ def test_get_by_url_fragment(self) -> None:
 class StoryCommitLogEntryModelUnitTest(test_utils.GenericTestBase):
     """Test the StoryCommitLogEntryModel class."""
 
+    def test_get_export_policy(self) -> None:
+        export_policy = story_models.StoryCommitLogEntryModel.get_export_policy() # pylint: disable=line-too-long
```

Comment:
To deal with this, do:

```
export_policy = (
    story_models.StoryCommitLogEntryModel.get_export_policy())
```

Always avoid pylint pragmas where possible.

Reply:
Done!